src/kv_client/mod.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
I think maybe this file is a rebasing error? I think it has moved to one of the crates